pages/common/sort.md

```diff
@@ -13,3 +13,7 @@
 - Sort passwd file by the 3rd field
 
 `sort -t: -k 3n /etc/passwd`
+
+- Sort file containing sizes in human readable form
+
+`sort -h {{filename}}`
```

Comment:
Also, `-h` is for Linux only. It does not work in OSX. In this case you can copy this page in linux and osx folder, removes the one in common, and add this line only to linux page.